### ISSUE TITLE
create 2d array of image examples

### ISSUE THREAD

**Issue by bnord** [deepskies/DeepBench#7]
This will replace the individual images that were in the overleaf. They should be much smaller.

**Comment by voetberg** [comment]
Second annoying ping of the day @bnord Opinion? on these images? We can rescale as needed. 
![image](https://github.com/deepskies/DeepBench/assets/27649232/e779235c-a0de-4927-9f69-69da349bc673)

**Comment by voetberg** [comment]
Resolved in the overleaf

**Comment by bnord** [comment]
Could I still make a few requests for changes for this plot? I forgot about this figure.

**Comment by voetberg** [comment]
Yeah sure! What are your requests?

**Comment by bnord** [comment]
My initial/hot-take ones are

1. could we label the columns?
2. could we increase the resolution significantly? e.g., much larger ppi
3. the pendulum plots are hard to understand. how can we make it clearer what's going on there?

**Comment by bnord** [comment]
could someone write a figure caption so that we can work on and talk about what we're planning to convey with the figure?

**Comment by voetberg** [comment]
Yep, there's already a caption - Copied from the overleaf 

```
Example outputs of **DeepBench**, containing shapes, astronomy objects, and the two pendulum implementations. Variants include a single object, a noisy single object, two objects, and two noisy objects. Pendulums show noisy and non-noisy variants of the Newtonian (left) and Hamiltonian (right) mathematical simulations.
```

I agree there should be a label on what's noisy and not, that one's easy(ish). I suppose the problem is going to be differentiating the hamiltonian and newtonian pendulums while differentiating the composition images

**Comment by bnord** [comment]
We can also make more than one figure.

I think the fundamental goal should be to appropriately visualize the major kinds (examples) of simulations DeepBench can make. And, then we can make whatever kinds of graphics are valuable for that.

**Comment by voetberg** [comment]
Sounds good. I was keeping it as one figure to save space but if we're not worried about it, we can certainly split it up. 

Minor update: 

![image](https://github.com/deepskies/DeepBench/assets/27649232/419dbe1e-7101-43de-95ef-7ad1adf33bc2)

![image](https://github.com/deepskies/DeepBench/assets/27649232/55dbcd8c-f4cb-49f1-a91f-f6aaae3afb07)

**Comment by bnord** [comment]
Re: Pendulum

What if we combine newtonian and hamiltonian or noise and noiseless? I think we should do one of those -- both for explainability and for efficient space usage.

**Comment by bnord** [comment]
For images, I think we should dial up noise for the noisy cases to make it obvious.

**Comment by bnord** [comment]
Could you add axis labels and units to all the plots?
For images, it could be pixel number, or it could be angular or physical units.
For the pendulums, I think it should be some angle or physical amplitude on y axis and time on the x axis?

**Comment by voetberg** [comment]
Yeah all of those are do-able! 

![image](https://github.com/deepskies/DeepBench/assets/27649232/9fee8e31-01d0-4fb3-9b9b-935ef4139e64)
The geometric and astro objects (they way they're coded) don't have a physical meaning beyond Pixels (mostly because all the values in the astro objects are relative to the size of the image.)

![image](https://github.com/deepskies/DeepBench/assets/27649232/55edf987-6e20-49f6-8e9c-563a8a486a18)

**Comment by bnord** [comment]
for the 2d images, let's try the following:
1. remove "example generated objects"
2. reduce white space around the image
3. remove "y pixels" and "x pixels"
4. label the tick marks numbers of pixels 
5. put the headings for each column at the top of each column

**Comment by bnord** [comment]
for the pendulums, let's try the following:
1. put the "angle" label on each y-axis
2. put tick marks with numbers on the y-axis of each panel
3. put tick marks with numbers on the x-axis of each panel
4. put the legend inside the border of one of the plots (it will apply to both)
5. remove "example generated physics simulations"

**Comment by voetberg** [comment]
You got it boss o7 
![image](https://github.com/deepskies/DeepBench/assets/27649232/9893d454-3534-46cf-97ff-6562e862a83c)
![image](https://github.com/deepskies/DeepBench/assets/27649232/899ce7e5-0ac8-4bf0-828c-9a334106b6f8)

**Comment by bnord** [comment]
these look a lot better. We're getting close.

**Comment by bnord** [comment]
for the 2d images, could we do the following?
1. put "pixel" or "pix" on the y axis and x axis labels
2. have the '0' on the x axis and y axis be the tick mark

**Comment by voetberg** [comment]
>  have the '0' on the x axis and y axis be the tick mark
Not sure what you mean by this? I swapped the direction of the y axis because I think it makes more sense but I'm not sure if that was what you were asking for

![image](https://github.com/deepskies/DeepBench/assets/27649232/a0b0b251-cfe9-4ae2-aa50-addd4851ebb4)


>put units on the x and y axis labels
I don't have units for the pendulums (it's not in the docs and I'm not sure.) I put in an issue for Omari and Becky to add them (as well as me adding in what I can for the astro objects). I'll ping them here as well @omaripaul57 @beckynevin 

![image](https://github.com/deepskies/DeepBench/assets/27649232/45bf0b23-cf9b-45d5-b02a-5a790133d3ab)

**Comment by voetberg** [comment]
@bnord Did we decide on these images?

**Comment by bnord** [comment]
Images:
1. put "pixel" on each x and y axis
2. "geometric" --> "geometry"
3. move "geometry" and "astronomy" to the left

Pendulum:
1. move legend so that it doesn't overlap with any data; change y axis range if necessary
2. change aspect ratio so to shrink the y axis and lengthen the x axis: this make it easier to see the patterns in the data.

**Comment by voetberg** [comment]
closed #136
